CHANGELOG

```diff
@@ -3,6 +3,19 @@ https://github.com/rtrlib/rtrlib/commits/master
 
 Release History:
 
+* Version 0.7.0
+    - Ensure shadow tables are only freed when initialized
+    - Fix SSH transport and host key verification
+    - Update tommyds implementation
+    - Fix check ensuring uniqueness of group preferences
+    - Fix race condition in rtr_stop
+    - Remove error PDU null termination check
+    - tools/rpki-rov: improve input validation
+    - Output log to stderr instead of stdout
+    - tools/rtrclient: several updates and refinements
+    - style: increase max line length to 120 characters
+    - tools/rtrclient: add roa export functionality
```

Comment:
be more concrete what is supported now

change order of items and put tools/ together.

Reply:
okay will adapt,